install/0001_00_cluster-version-operator_03_service.yaml

```diff
@@ -1,16 +1,19 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: cluster-version-operator
+  name: metrics
   namespace: openshift-cluster-version
   labels:
     k8s-app: cluster-version-operator
   annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: openshift-cluster-version-serving-cert
     exclude.release.openshift.io/internal-openshift-hosted: "true"
 spec:
   type: ClusterIP
   selector:
     k8s-app: cluster-version-operator
   ports:
-  - name: metrics
-    port: 9099 # chosen to be in the internal open range
+  - name: https
+    port: 443
+    protocol: TCP
+    targetPort: 8443
```

Comment:
Isn't HTTPS sufficient?  I don't see why we'd need to expose TCP too.

Reply:
> Isn't HTTPS sufficient...

Scratch this, reading the Service docs, I see that `protocol: TCP` is [the default][1].  I'm fine with making that default explicit.  `targetPort` will need to match up with something that the CVO is actually listening on, though (currently [9099 by default][1]).

[1]: https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
[2]: https://github.com/openshift/cluster-version-operator/blob/ac394f1f73a6e1367d0af8622d4f9af545a6e39a/pkg/start/start.go#L86

Reply:
Fixed as of 16da75689.